README.md

```diff
@@ -80,7 +80,7 @@ A sane set of defaults exists in `.env.example`
 
 `flask db upgrade`
 
-15. Run the service
+15. Run the service, if also running the Admin locally you may want to default to another port such as 6013
```

Comment:
This is not necessary - the default suggested values for admin and api are 6012 and 6011 respectively.

Reply:
Not sure how I missed that it was 6011 for the api. For some reason I saw them both at 6012. I'm removing this since it makes no sense :)